homeassistant/components/whirlpool/climate.py

```diff
@@ -104,14 +104,22 @@ class AirConEntity(ClimateEntity):
     _attr_target_temperature_step = SUPPORTED_TARGET_TEMPERATURE_STEP
     _attr_temperature_unit = UnitOfTemperature.CELSIUS
     _attr_should_poll = False
+    _attr_has_entity_name = True
 
     def __init__(self, hass, said, name, backend_selector: BackendSelector, auth: Auth):
         """Initialize the entity."""
         self._aircon = Aircon(backend_selector, auth, said, self.async_write_ha_state)
 
         self.entity_id = generate_entity_id(ENTITY_ID_FORMAT, said, hass=hass)
-        self._attr_name = name if name is not None else said
+        self._attr_name = None
```

Comment:
If this is statically defined, it can be moved into the class variables instead.

Reply:
I don't think I need to set it at all (nor should I?) It's None in the base

Reply:
Right, even better 👍